framework/platform/platform.h

```diff
@@ -100,6 +100,11 @@ class Platform
 
 	Window &get_window() const;
 
+	const std::unique_ptr<Window> &get_window_ptr() const
```

Comment:
Why does this function need to be added? We already have `Window &get_window() const;`, which invalidates the checklist item `I have used existing framework/helper functions where possible`

Reply:
If you have an argument for adding this, can we place implementation inside the `.cpp`

Reply:
will remove